.gitignore

```diff
@@ -0,0 +1 @@
+tmp
```

Comment:
I prefer to make gitignore entries very specific so that they don't catch entries anywhere in the hierarchy. Thus:

```
/tmp/
```